docs/index.rst

```diff
@@ -20,6 +20,8 @@ The configuration of the *Kafka* plugin is very simple, consisting of four param
 
   - **Send JSON**: This controls how JSON data points should be sent to Kafka. These may be sent as strings or as JSON objects.
 
+  - **Compression Type**: Type of compression codec to be used before sending data to kafka broker. Supported compression types are gzip, snappy and lz4. Default value is none.
```

Comment:
Kafka

Also, do we need any runtime installation for this on FogLAMP instance machine?

Reply:
Not required... dependencies are already in place.

Reply:
I suspect Type is not really needed, it is merely 'Compression Codec". 

I would also make the text "The compression codec to be used to send data to the Kafka broker. Supported compression codecs are; gzip, snappy, lz4 or none. The default value is none, in which case no compression will take place."